src/apps/api/views/sample.py

```diff
@@ -299,17 +299,13 @@ def get_samples_by_sequencing_centre(request):
         out = filter_for_API(samples)
     return finish_request(out)
 
-def get_updatable_fields_by_project(request, project):
-    project_lst = project.split(",")
-    project_lst = list(map(lambda x: x.strip(), project_lst))
-    # remove any empty elements in the list (e.g. where 2 or more comas have been typed in error
-    project_lst[:] = [x.lower() for x in project_lst if x] # Convert all strings in the list to lowercase
-    out = list()
-    
-    for project in project_lst:
-        if project in lookup.DTOL_NO_COMPLIANCE_FIELDS:
-            out.append({project.upper(): lookup.DTOL_NO_COMPLIANCE_FIELDS[project]})
-    return finish_request(out)
+def get_updatable_fields_by_project(request):
+    project = request.GET.get('project', str()).lower()
+    non_compliant_fields = d_utils.get_non_compliant_fields(component='sample', project=project, can_be_exported=True)
```

Comment:
why we need "can_be_exported = true"?  It is not going to export the field value, but to tell user which field can be updated. I think there is nothing to do "export" here.